src/apiCalls.js

```diff
@@ -4,10 +4,8 @@ const allMoviesData = () => {
 }
 
 const singleMovieData = (id) => {
-  // console.log("ID", id)
   return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
   .then(response => response.json())
```

Comment:
This is where it would be nice to do some error handling to double-check that the response is a success (eg. status code 200). Check out `response.ok` to help you out with this.